src/main/java/net/dries007/tfc/common/blocks/TFCBlocks.java

```diff
@@ -72,6 +74,7 @@
 
     public static final RegistryObject<SnowPileBlock> SNOW_PILE = register("snow_pile", () -> new SnowPileBlock(new ForgeBlockProperties(Properties.copy(Blocks.SNOW)).tileEntity(SnowPileTileEntity::new)), EARTH);
     public static final RegistryObject<CalciteBlock> CALCITE = register("calcite", () -> new CalciteBlock(Properties.of(Material.GLASS).noDrops().instabreak().sound(SoundType.GLASS)));
+    public static final RegistryObject<CalciteBlock> ICICLE = register("icicle", () -> new CalciteBlock(Properties.of(Material.ICE).noDrops().instabreak().sound(SoundType.GLASS).noOcclusion()));
```

Comment:
Should rename `CalciteBlock` then, given it's not just for calcite. `ThinSpikeBlock` imo.